pytorch_lightning/utilities/exceptions.py

```diff
@@ -27,3 +27,7 @@ class ExitGracefullyException(SystemExit):
     This signals Lightning to try to create a fault-tolerance checkpoint once the current batch or epoch is reached
     (assuming it can be done under 30 sec). After the checkpoint is saved, Lightning will exit.
     """
+
+
+class _TunerExitException(Exception):
```

Comment:
why is this private when all other exceptions are public?

Reply:
this is not something that is user-facing. Also trying to avoid it being documented inside API references.